### DISCUSSION TITLE
Intermediate page

### DISCUSSION THREAD

**Discussion by Kolkapetkinsyn** [YOURLS/YOURLS#3528] [Plugins]
Hi guys. Is there a plugin for organizing an intermediate page? So that when you click on the link, the information page is shown first, and only then the landing page is shown.

The question was formulated with the help of an interpreter, if the question is not clear, ask again.

**Comment by dgw** [comment]
How about [the Preview URL example](https://yourls.org/docs/development/examples/preview.html)?
